test/test_control_client.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
```

Comment:
True. I'm almost certain this came in as a copy paste error when I inserted the license headers.